spec/halogen/links_spec.rb

```diff
@@ -68,6 +68,42 @@
   end
 
   describe Halogen::Links::InstanceMethods do
+    describe "options[:include_links]" do
```

Comment:
there is no linter, but we should use `'` since the rest of the specs do as well